### PR TITLE
Update feedback component

### PR DESCRIPTION
**Pull request by sam-warren**
# Fixes or Implements [AB#9013](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9013)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Changed title and description of feedback component.

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### UI Changes

YES

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

**Review by leobcgov — CHANGES_REQUESTED** [review]
@sam-warren it seems that you may need to fix the lint. Here are the build's errors:
```
/opt/az/_work/3/s/Apps/WebClient/src/ClientApp/src/views/login.vue
     7:44  error  Delete `;`  prettier-vue/prettier
    22:44  error  Delete `;`  prettier-vue/prettier
```

**Review by leobcgov — APPROVED** [review]

**Review by leobcgov — CHANGES_REQUESTED** [review]
btw, per team discussion yesterday, we need to wait for other PRs for PROD deployment before merging these new PRs to Dev. I will approve this PR later to make sure we don't merge this to Dev branch by accident then.

**Review by leobcgov — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/login.vue
Comment (ninosamson) [inline Apps/WebClient/src/ClientApp/src/views/login.vue]: why the semicolon sam?

**Review by sslaws — APPROVED** [review]

**Review by FuriousLlama — APPROVED** [review]

**Review by ninosamson — APPROVED** [review]

**Pull request merged**
